enclave/core/tracee.c

```diff
@@ -84,13 +149,6 @@ oe_result_t oe_log(oe_log_level_t level, const char* fmt, ...)
         goto done;
     }
 
-    // Check if this message should be skipped
```

Comment:
why is this deleted?

Reply:
This is moved into oe_log_enclave_message(). This check is relevant only when callback is not set.